packages/react-components/src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -182,3 +198,7 @@ export const RichTextEditor = React.forwardRef<RichTextEditorComponentRef, RichT
     </div>
   );
 });
+
+const isEmpty = (value?: string): boolean => {
```

Comment:
Do we share something like this with SendBox, that could go in the shared utils?

Reply:
Deleted it :) because the undefined weren't recognized correctly if the function was used